sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/data_asset.py

```diff
@@ -13,15 +13,10 @@
 class DataAsset:
     """Data Asset entity
 
-    :param data_id: Arm id of registered data asset
-    :param data_id: str
-    :param name: Name of data asset
-    :type name: str
-    :param path: Path where the data asset is stored.
-    :type path: str
-    :param version: Version of data asset.
-    :type version" int
-
+    :ivar str data_id: Arm id of registered data asset
+    :ivar str name: Name of data asset
+    :ivar str path: Path where the data asset is stored.
+    :ivar int version: Version of data asset.
```

Comment:
Could we make these constructor params keyword-only while we're here? :)
(You can also remove `kwargs` from the constructor - it isn't needed for models).

Reply:
Should be resolved by https://github.com/Azure/azure-sdk-for-python/pull/34814/commits/587fa9933daafbb42997f0e395efafd2234cd894 and https://github.com/Azure/azure-sdk-for-python/pull/34814/commits/68bd5734af17b4d79bfceda4c114c587be59f6e4